config/environments/production.rb

```diff
@@ -1,5 +1,3 @@
-require Rails.root.join("config", "secret") if Rails.root.join("config", "secret.rb").file?
```

Comment:
Do we have `config/secret.rb` file in production? Running `rails app:update` removed it and I didn't add it back because there is no such file in repo.